test/util/IdTableHelpers.h

```diff
@@ -56,6 +56,17 @@ class CopyableIdTable : public TableImpl<N> {
 using IntOrId = std::variant<int64_t, Id>;
 using VectorTable = std::vector<std::vector<IntOrId>>;
 
+// Helper: construct a single-column VectorTable containing the inclusive
+// integer range [a, b]. If a > b, returns an empty table.
+static inline VectorTable makeRangeVectorTable(size_t a, size_t b) {
```

Comment:
Why the `inclusive [a, b]` , in C++ the convention is to have the non-exclusive `[a, b)`, is that important for something, otherwise please stick to the convention.